README.md

```diff
@@ -1,19 +1,24 @@
-[![Build Status](https://github.com/debezium/container-images/actions/workflows/image-builds-auth.yml/badge.svg?branch=main)](https://github.com/debezium/container-images/actions/workflows/image-builds-auth.yml)
-[![License](http://img.shields.io/:license-mit-brightgreen.svg)](https://opensource.org/licenses/MIT)
-[![Quay.io](https://img.shields.io/badge/quay.io-images-brightgreen)](https://quay.io/organization/debezium)
+# Debezium Docker Images
 
-This repository contains the primary container images for Debezium, and they are automatically built and published on [Quay.io](https://quay.io/organization/debezium).
+This is a forked version of https://github.com/debezium/container-images.
 
-# What is Debezium?
+We forked it for 2 reasons:
 
-Debezium is a distributed platform that turns your existing databases into event streams, so applications can quickly react to each row-level change in the databases. Debezium is built on top of Kafka and provides Kafka Connect compatible connectors that monitor specific database management systems. Debezium records the history of data changes in Kafka logs, so your application can be stopped and restarted at any time and can easily consume all of the events it missed while it was not running, ensuring that all events are processed correctly and completely.
+1. Inject AWS MSK IAM dependency
+2. Remove jboss user (remove the need for initContainers in K8s)
 
-Debezium is open source under the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.html)
+Only certain versions are actually published to Dockerhub.
```

Comment:
@Tang8330 Just as a side note, all newer images can be found on quay.io: https://quay.io/organization/debezium , some older versions are only available on Dockerhub.
Which images were you missing?